cmd/internal/switcher/cumulus.go

```diff
@@ -0,0 +1,53 @@
+package switcher
+
+import (
+	"fmt"
+	"net"
+	"strings"
+
+	"go.uber.org/zap"
+)
+
+type Cumulus struct {
+	frrApplier        *FrrApplier
+	interfacesApplier *InterfacesApplier
+	log               *zap.SugaredLogger
+}
+
+func NewCumulus(log *zap.SugaredLogger, frrTplFile, interfacesTplFile string) *Cumulus {
```

Comment:
Kind of taste, but wouldnt it be more readable if a core.NOS is returned here ?

Reply:
According to the Principle of Least Surprise, I followed the Go convention "Accept interfaces, return structs".